lib/scala/kg/src/main/scala/io/github/tetherlessworld/mcsapps/lib/kg/stores/postgres/PostgresKgCommandStore.scala

```diff
@@ -0,0 +1,81 @@
+package io.github.tetherlessworld.mcsapps.lib.kg.stores.postgres
+
+import com.google.inject.Inject
+import io.github.tetherlessworld.mcsapps.lib.kg.formats.kgtk.KgtkEdgeWithNodes
+import io.github.tetherlessworld.mcsapps.lib.kg.models.edge.KgEdge
+import io.github.tetherlessworld.mcsapps.lib.kg.models.node.KgNode
+import io.github.tetherlessworld.mcsapps.lib.kg.models.path.KgPath
+import io.github.tetherlessworld.mcsapps.lib.kg.models.source.KgSource
+import io.github.tetherlessworld.mcsapps.lib.kg.stores.{KgCommandStore, KgCommandStoreTransaction}
+import play.api.db.slick.DatabaseConfigProvider
+
+import scala.concurrent.ExecutionContext
+
+class PostgresKgCommandStore @Inject()(dbConfigProvider: DatabaseConfigProvider)(implicit executionContext: ExecutionContext) extends AbstractPostgresKgStore(dbConfigProvider) with KgCommandStore {
+  import profile.api._
+
+  private final class PostgresKgCommandStoreTransaction extends KgCommandStoreTransaction {
```

Comment:
May be able to do 1 Postgres transaction : 1 KgCommandStoreTransaction, which would be ideal. neo4j couldn't do that because it buffered transactions in memory. Postgres doesn't do that.

Reply:
Gotcha, I will try to do that